src/Microsoft.OpenApi.Tool/OpenApiService.cs

```diff
@@ -35,12 +35,20 @@ static class OpenApiService
 
```

Comment:
Defensive programming for the params?

Reply:
The `version` and `format` params have default values, so they can't equate to null at any point.
`operationIds` is nullable since its an optional filtering parameter.